### DISCUSSION TITLE
米家门锁E10识别无法识别离家模式

### DISCUSSION THREAD

**Discussion by xpfyg** [XiaoMi/ha_xiaomi_home#357] [Ideas]
<img width="1105" alt="image" src="https://github.com/user-attachments/assets/0a6f9b5e-2e45-4207-829d-de2b07cbcece" />
小米门锁触发离家模式，只会显示为触发事件，无法识别是离家模式

**Comment by JimmyKmi** [comment]
请检索设备是否支持离家模式上报：https://mijia.wiki/products
